src/ansi_color_codes.h

```diff
@@ -0,0 +1,79 @@
+#ifndef ANSI_COLOR_CODES_H_
```

Comment:
In some local changes I just stole the `vt.h` header from OoT decomp. I like the `VT_` prefix those have since I think `reset` in particular is too general of a name.

Reply:
I don't feel like copying oot decomp's vt.h since
- afaict (wikipedia) "VT" only refers to an implementation of ansi color codes
- it's less complete
- afaict `VT_COLOR_LIGHTGRAY` and `VT_COLOR_DARKGRAY` aren't actual codes

Taking the good things from vt.h though (color names, using a prefix) I did improve ansi_color_codes.h (agreed on no-prefix/"reset" being too generic)